ethcore/src/snapshot/service.rs

```diff
@@ -618,6 +638,13 @@ impl Drop for Service {
 	}
 }
 
+pub struct NoopDBRestore;
```

Comment:
add a doc comment

Reply:
just saw this. added doc comments to that struct and the following trait. 

was planning to move the split-out test classes shared across crates to a central `parity-test` crate. think this would be a worthwhile change, but wanted to consult the team before starting in on the work. what are your thoughts on taking this approach?